src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -31,6 +31,7 @@ module SoilBiogeochemCarbonFluxType
      real(r8), pointer :: decomp_cascade_ctransfer_vr_col           (:,:,:) ! vertically-resolved C transferred along deomposition cascade (gC/m3/s)
      real(r8), pointer :: decomp_cascade_ctransfer_col              (:,:)   ! vertically-integrated (diagnostic) C transferred along decomposition cascade (gC/m2/s)
      real(r8), pointer :: cn_col                                    (:,:)   ! (gC/gN) C:N ratio by pool
+     real(r8), pointer :: litr_lig_c_to_n_col                       (:)     ! (gC/gN) lignin C:N ratio; FATES equivalent of cnveg_carbonflux_inst%ligninNratioAvg_col because FATES doesn't have access to cnveg_carbonflux_inst; could reduce duplication by consolidating cnveg_carbonflux_inst%ligninNratioAvg_col into this one
```

Comment:
What I want to try before we merge this code is reduce the duplication explained in the above comment.

Reply:
The next commit handles the duplication reduction successfully. 
The CTSM-MIMICS test passes, including compared to the baseline. 
The CTSM-MIMICS-FATES test passes.

I have submitted the test suites and will report results in the morning.

Reply:
A short list of tests FAIL on cheyenne and izumi, so I need to investigate.